isb_web/main.py

```diff
@@ -103,6 +104,7 @@ async def thing_list_types(
 async def get_thing(
     identifier: str,
     full: bool = False,
+    format: str = "original",
```

Comment:
Can these parameters be typed to an enumeration of String values?  Do we care?

Reply:
They should be (makes the api clearer). Not sure exactly how to do that in fastapi, probably need to create a class of enum or something

Reply:
Good, I'll investigate on Friday.